mathlibtools/lib.py

```diff
@@ -149,7 +149,7 @@ def get_mathlib_archive(rev: str, url:str = '', force: bool = False,
     """
 
     # we check for xz archives first
-    fnames = [rev + '.tar.xz', rev + '.tar.gz', rev + '.tar.bz2']
```

Comment:
@robertylewis Did you also migrate the `tar.gz` archives to `tar.xz`?

Reply:
Yes, every archive should exist in both forms right now.